src/rpc/multipass.proto

```diff
@@ -44,6 +44,7 @@ message LaunchRequest {
     string time_zone = 7;
     string cloud_init_user_data = 8;
     string remote_name = 9;
+    int32 opt_in_reply = 10;
```

Comment:
This should be an enum that mirrors mp::OptInStatus, this way there's no ambiguity which values can be set here.